sched_slack_bot/model/schedule.py

```diff
@@ -144,13 +145,16 @@ def from_modal_submission(cls, submission_body: SlackBody) -> Schedule:
             state=state, date_input_block_ids=get_datetime_block_ids(label=SECOND_ROTATION_LABEL)
         )
 
-        time_between_rotations = second_rotation - next_rotation
+        time_between_rotations = abs(second_rotation - next_rotation)
 
         schedule_id = submission_body["view"]["external_id"]
 
-        if schedule_id == CREATE_NEW_SCHEDULE_VIEW_ID:
+        if schedule_id.startswith(CREATE_NEW_SCHEDULE_VIEW_ID):
             schedule_id = str(uuid.uuid4())
 
+        if SHEDULE_VIEW_ID_SCHEDULE_ID_DELIMITER in schedule_id:
+            schedule_id = schedule_id.split(SHEDULE_VIEW_ID_SCHEDULE_ID_DELIMITER)[0]
```

Comment:
Are these really all uses of `external_id`?

Reply:
I think so: there is literally nowhere else it is used, can you elaborate what makes you unsure?

Reply:
I just double-checked. I think we're fine 👍